symplyphysics/laws/thermodynamics/average_square_of_velocity.py

```diff
@@ -0,0 +1,30 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+                           validate_input, validate_output)
+
+# Description
+## For an ideal gas, the average square of velocity is directly proportional to its temperature and inversely proportional to the molar mass of the gas: <V^2> = 3 k T / m
+## Where:
+## <V^2> - average square of velocity
+## T - temperature in gas
+## m - mass of a gas
```

Comment:
Is it mass of gas or molar mass?

Reply:
The mass of the gas. I was wrong last time.

Reply:
I'm not sure it is correct.

Reply:
I'm wrong again. I'll fix it. The mass for one atom. This is derived from the molar mass and Avogadro constant.

Reply:
Mass of molecule, I think.

Reply:
Yes: $m = \mu / N_A$.